float32[2] map_dimensions
uint8 n_obs
uint8 n_hip
Obstacle[] obstacles
Hiperboloid[] hiperboloids